modules/internal/ArrayViewReindex.chpl

```diff
@@ -26,6 +26,268 @@ module ArrayViewReindex {
   use ChapelStandard;
 
   //
+  // This class represents a distribution that knows how to create
+  // reindexed domains and arrays similar to the one that caused
+  // it to be created.  It stores a pointer down to the distribution
+  // that it creates lower-dimensional views of, and two tuples
+  // indicating which dimensions were collapsed by the reindexed
+  // and what the indices of those dimensions are.
```

Comment:
This comment mentions collapsed dimensions, was this mistakenly copy/pasted from the rank-change array view?